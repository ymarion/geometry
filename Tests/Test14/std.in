# Initializing an object
C c1 0 0 10
# Saving it
SAVE testOut.txt

