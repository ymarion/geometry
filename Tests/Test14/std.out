# Creations
OK
OK
OK
OK
OK
OK
# SAVE
OK
